docker-compose.yml

```diff
@@ -24,7 +24,7 @@ services:
   #             https://selenium-python.readthedocs.io/api.html#module-selenium.webdriver.common.proxy
   #
   #       Alternative Playwright URL, do not use "'s or 's!
-  #      - PLAYWRIGHT_DRIVER_URL=ws://playwright-chrome:3000&stealth=1&--disable-web-security=true
+  #      - PLAYWRIGHT_DRIVER_URL=ws://playwright-chrome:3000/&stealth=1&--disable-web-security=true
```

Comment:
```suggestion
  #      - PLAYWRIGHT_DRIVER_URL=ws://playwright-chrome:3000/&stealth=1&--disable-web-security=true
  #      - PLAYWRIGHT_DRIVER_URL=ws://playwright-chrome:3000/?stealth=1&--disable-web-security=true
  
```

Reply:
thanks! better again is to make sure `?` is at the start, if you change this, i'll merge the PR, thanks!

Reply:
Sorry, I was busy during the last day. Thanks for fixing it in the meantime.